staging/src/k8s.io/client-go/util/jsonpath/jsonpath_test.go

```diff
@@ -272,6 +334,18 @@ func TestStructInput(t *testing.T) {
 		{"recurarray", "{..Book[2]}", storeData,
 			`{"Category":"fiction","Author":"Herman Melville","Title":"Moby Dick","Price":8.99}`, false},
 		{"bool", "{.Bicycle[?(@.IsNew==true)]}", storeData, `{"Color":"red","Price":19.95,"IsNew":true}`, false},
+		{"objects", "{.Labels[*]}", storeData, "10 15 20", false},
+		{"objects", "{$.Employees[*]}", storeData, "manager clerk", false},
+		{"empty", "{.Name[*]}", storeData, "", true},
+		{"nested-map", "{.Items[*]}", storeData, "{\"notebooks\":\"5\",\"pencils\":\"1\",\"pens\":\"2\"} {\"pulses\":\"3\",\"rice\":\"1\"} {\"bike\":{\"color\":{\"color\":\"red\",\"texture\":\"shinning\"},\"wheels\":{\"company\":\"any\",\"quantity\":\"2\"}},\"car\":{\"color\":{\"color\":\"silver\",\"texture\":\"shinning\"},\"wheels\":{\"company\":\"any\",\"quantity\":\"4\"}}}", false},
```

Comment:
quote the output like this to avoid needing to \-escape quotes:
```
`{"notebooks":...`
```

Reply:
Done. Thank you for pointing out. The formatting was done by the IDE, and hence went unnoticed.